snippets/cpp/VS_Snippets_CLR/conceptual.interop.marshaling/cpp/structures.cpp

```diff
@@ -33,11 +33,11 @@ public value struct MyArrayStruct
 {
 public:
     bool flag;
-    [MarshalAs(UnmanagedType::ByValArray, SizeConst=3)]
+    [MarshalAs(UnmanagedType::ByValArray, SizeConst = 3)]
     array<int>^ vals;
 };
 
-public ref class LibWrap
+private ref class NativeMethods
```

Comment:
I may be looking at the diffs wrong, but shouldn't this be `internal` instead of `private`?